CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
This should use FALSE or NO as the default. See the CMake docs for [if](https://cmake.org/cmake/help/v3.0/command/if.html).